pages/index.js

```diff
@@ -1,3 +1,10 @@
+import Map from "../components/Map/Map";
+
 export default function Home() {
-  return <h1>Home</h1>;
+  return (
+    <div>
+      <h1>Home</h1>
+      <Map></Map>
```

Comment:
minor detail: you could use a self-closing tag to indicate that this component can't have child components
```suggestion
      <Map />
```

Reply:
done